microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -820,6 +1105,89 @@ Failed:
 ---
 
 
+<!-- ====================================================================== -->
+## Shared buffer
+
+<!-- section from RelNotes 111 -->
+
+The SharedBuffer API supports sharing buffers between the WebView2 host app process and WebView2 renderer process, based on shared memory from the OS.
+
+<!-- todo: link to JavaScript Ref?  is there an article to cross-link?
```

Comment:
@peiche-jessica @novac42  - un-comment this JS Ref link?  else keep the hidden comment but clean it up